app/forms/MultiYearPartnersIncomeQuestionForm.scala

```diff
@@ -24,25 +24,31 @@ import play.api.data.{Form, FormError}
 
 object MultiYearPartnersIncomeQuestionForm {
 
-  private val PRE_ERROR_KEY = ("pages.form.field-required.")
+  private val PRE_ERROR_KEY = "pages.form.field-required."
 
   implicit def requiredBooleanFormatter: Formatter[Boolean] = new Formatter[Boolean] {
     override val format = Some(("format.boolean", Nil))
 
     def bind(key: String, data: Map[String, String]) = {
-      Right(data.get(key).getOrElse("")).right.flatMap {
+      val isScottish: Boolean = data.get("is-scottish").getOrElse("false").toBoolean
```

Comment:
Perhaps we should add some validation to make sure the string is a boolean?

Reply:
changed to match case to cover any value to boolean